app/code/Transoft/Blog/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="catalog" translate="label" type="text" sortOrder="40" showInDefault="1" showInWebsite="1" showInStore="1">
+        <class>separator-top</class>
+        <label>Catalog</label>
+        <tab>catalog</tab>
+        <resource>Magento_Catalog::config_catalog</resource>
```

Comment:
Please do not copy the existing section data

Reply:
How do i override existing section in this case?

Reply:
> How do i override existing section in this case?

I didn't say **override**. I said **copy**. **ALL** xml files ( config, acl, di, etc. ) are merged. So your code should look like:
```
<system>
    <section id="catalog">
		<group id="blog" translate="label" type="text" sortOrder="90" showInDefault="1" showInWebsite="1" showInStore="1">
		...
```

Reply:
fixed